jobs/jobs/utils.py

```diff
@@ -583,3 +586,54 @@ def get_taxonomy_links(
         )
         for category_link in categories_links
     ]
+
+
+@alru_cache(maxsize=256)
```

Comment:
What is username will change? 
As I can see, this function doesn't implement any cache invalidation handlers

Reply:
Current solution does not have any scenario in case username changes and doesn't have any cache invalidations.
If we imply that username could be changed and we have to handle it - it means no caching possible.

Reply:
Also, 'users' API does not support changing any attributes of any user, including username.
But it is still possible for an admin in Keycloak directly.
I am not sure whether we should keep in mind this possibility

Reply:
Deleted